python/pyspark/mllib/stat/_statistics.py

```diff
@@ -16,51 +16,57 @@
 #
 
 import sys
+from typing import overload, List, Optional, Union
+from typing_extensions import Literal
```

Comment:
We should never have `typing_extensions` extensions imported directly in  `py` module. Let's move this and `CorrelationMethod` to `_typing.pyi`.

Reply:
thanks! updated. I also move 
```python
DistName = Literal["norm"]
```
to ```_typing.pyi```